CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.1.0 (2019-08-30)
+
+- Added `preserveWhiteSpace` property to `SkyConfirmConfig` to allow consumers to preserve whitespace and new lines inside the confirm component. [#65](https://github.com/blackbaud/skyux-modals/pull/65) (Thanks [@Blackbaud-ShaydeNofziger](https://github.com/Blackbaud-ShaydeNofziger)!)
+- Fixed text from being displayed vertically in IE11 when select field components and list view checklist components are inside a modal. [#64](https://github.com/blackbaud/skyux-modals/pull/64)
```

Comment:
I think we should generalize this and just say "when the modal content is wider than the modal body".

Reply:
That makes sense to me. Also, "Fixed text from..." reads awkwardly. We aren't fixing text. We are fixing the component. Also, I'm not sure what we mean here by displaying text vertically; can we just be vague and say that the fix is to display text correctly?. Taking all these suggestions into account, I'd change this to "Fixed the modal component to display text correctly in IE11 when modal content is wider than the modal body."